### PR TITLE
pythonPackages.clickhouse-driver: init at 0.1.5

### PR DESCRIPTION
**Pull request by breakds**
###### Motivation for this change

There is already [clickhouse](https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/services/databases/clickhouse.nix) packaged in nixpkgs, and it would more convenient for the big data developers if the various clients are there as well. This pull request packages [clickhouse-driver](https://pypi.org/project/clickhouse-driver/), which enables python developers to interact with a clickhouse server.

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/clickhouse-cityhash/default.nix [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0z8nl0ly2p1h6nygwxs6y40q8y424w40fkjv3jyf8vvcg4h7sdrg";
+  };
+
+  propagatedBuildInputs = [ setuptools ];
+
+  doCheck = true;
+  pythonImportTests = [ "clickhouse_cityhash" ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/clickhouse-cityhash/default.nix:19]:
```suggestion
  pythonImportsCheck = [ "clickhouse_cityhash" ];
```

File: pkgs/development/python-modules/clickhouse-driver/default.nix [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  ];
+
+  checkInputs = [
+    freezegun
+    mock
+    nose
+  ];
+
+  doCheck = true;
+  pythonImportTests = [ "clickhouse_driver" ];
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/clickhouse-driver/default.nix:40]:
```suggestion
  pythonImportsCheck = [ "clickhouse_driver" ];
```

**Review by SuperSandro2000 — CHANGES_REQUESTED** [review]

**Comment by SuperSandro2000** [comment]
Please put each package into its own commit with the message like ``pythonPackages.clickhouse-driver: init at 0.1.5``.

File: pkgs/development/python-modules/clickhouse-cityhash/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  version = "1.0.2.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0z8nl0ly2p1h6nygwxs6y40q8y424w40fkjv3jyf8vvcg4h7sdrg";
+  };
+
+  propagatedBuildInputs = [ setuptools ];
+
+  doCheck = true;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/clickhouse-cityhash/default.nix:18]:
```suggestion
  doCheck = false;
```

This package does not contain any tests.

File: pkgs/development/python-modules/clickhouse-cityhash/default.nix (hunk 1 above)
Comment (breakds) [reply pkgs/development/python-modules/clickhouse-cityhash/default.nix]: Fixed. Thanks!

**Comment by breakds** [comment]
> Please put each package into its own commit with the message like `pythonPackages.clickhouse-driver: init at 0.1.5`.

Split the original commit into two for each packages. Thanks for prompt review!

**Comment by RaghavSood** [comment]
Result of `nixpkgs-review pr 106039` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python37Packages.clickhouse-cityhash</li>
    <li>python37Packages.clickhouse-driver</li>
    <li>python38Packages.clickhouse-cityhash</li>
    <li>python38Packages.clickhouse-driver</li>
    <li>python39Packages.clickhouse-cityhash</li>
    <li>python39Packages.clickhouse-driver</li>
  </ul>
</details>

**Review by RaghavSood — APPROVED** [review]

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106039` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python37Packages.clickhouse-cityhash</li>
    <li>python37Packages.clickhouse-driver</li>
    <li>python38Packages.clickhouse-cityhash</li>
    <li>python38Packages.clickhouse-driver</li>
    <li>python39Packages.clickhouse-cityhash</li>
    <li>python39Packages.clickhouse-driver</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 106039` run on x86_64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>6 packages built:</summary>
  <ul>
    <li>python37Packages.clickhouse-cityhash</li>
    <li>python37Packages.clickhouse-driver</li>
    <li>python38Packages.clickhouse-cityhash</li>
    <li>python38Packages.clickhouse-driver</li>
    <li>python39Packages.clickhouse-cityhash</li>
    <li>python39Packages.clickhouse-driver</li>
  </ul>
</details>

**Pull request merged**